### PR TITLE
CACTUS-600 :: Flexible Icon Sizes

### PR DESCRIPTION
**Pull request by mikesoltow**
Ticket and UAT: https://repayonline.atlassian.net/browse/CACTUS-600

Not much to add onto the testing description on the ticket.  Just be sure to inspect the CSS to make sure that it's still respecting the theme sizes when you use tiny/small/medium/large.

### CODE REVIEW

File: modules/cactus-icons/stories/Icons.story.tsx [hunk 1]
```diff
@@ -21,16 +21,11 @@ export const One = (): ReactElement => {
...
-      iconSize={select('iconSize', iconSizes, 'large')}
-      style={{ color: theme.colors.callToAction }}
-    />
-  )
+  return <Icon iconSize={text('iconSize', 'large')} style={{ color: theme.colors.callToAction }} />
 }
 
 export const All = (): ReactElement => {
-  const size = select('iconSize', iconSizes, 'large')
+  const size = test('iconSize', 'large')
```
Comment (daniloPenaR) [inline modules/cactus-icons/stories/Icons.story.tsx:28]: Small error there, this should be `text` I guess 😅

File: modules/cactus-icons/svgr.config.js [hunk 2]
```diff
@@ -58,7 +57,7 @@ export default COMPONENT_NAME
     placeholderPattern: /^[_A-Z]{2,}$/,
   })
 
-  return typeScriptTpl({ JSX: jsx, COMPONENT_NAME: componentName, TS_IGNORE: '// @ts-ignore' })
+  return typeScriptTpl({ JSX: jsx, COMPONENT_NAME: componentName })
 }
 
 module.exports = {
```
Comment (daniloPenaR) [inline modules/cactus-icons/svgr.config.js:63]: Out of curiosity, How are we using this file and this template?

File: modules/cactus-icons/stories/Icons.story.tsx (hunk 1 above)
Comment (mikesoltow) [reply modules/cactus-icons/stories/Icons.story.tsx]: Haha, thanks for catching this!

File: modules/cactus-icons/svgr.config.js (hunk 2 above)
Comment (mikesoltow) [reply modules/cactus-icons/svgr.config.js:63]: This file is used by `scripts/svgr.js` to generate `.tsx` files/components for each of our icons using a tool called [SVGR](https://react-svgr.com/).  In particular, we use the [custom templates](https://react-svgr.com/docs/custom-templates/) functionality of SVGR.

**Review by daniloPenaR — APPROVED** [review]

**Review by Dhalton — APPROVED** [review]

**Pull request merged**